pybricksdev/cli/__init__.py

```diff
@@ -280,21 +281,38 @@ async def reconnect_hub():
                     response = await hub.race_disconnect(
                         hub.race_power_button_press(
                             questionary.select(
-                                "Would you like to re-compile your code?",
+                                f"Would you like to re-compile {os.path.basename(args.file.name)}?",
                                 response_options,
                                 default=(
-                                    response_options[0]
+                                    response_options[1]
```

Comment:
Hmm... perhaps we should introduce an `IntEnum` for these array index values so that we only have to change the values in one place if we add more menu items in the future.